releases/v0.4.0.md

```diff
@@ -0,0 +1,67 @@
+# Release Notes for v0.4.0
+Release Date: 2023-03-03 
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+- Skopio and umoci dependencies are removed with our image-rs component fully integrated
+- //add more new stuff here
```

Comment:
Should we add something about caching for the CI? It is now supported for SEV. Not sure it's part of the release exactly, though.